src/main/resources/project/jython/stop_cluster.jython

```diff
@@ -38,15 +39,46 @@ clusterName = r'''
 [% cluster_name %]
 '''.strip()
 
+timeout = r'''
+[% timeout %]
```

Comment:
Why not shutdownTimeout, like the parameter name? It's not very convenient, to remember all the names.